buildSrc/src/main/java/app/tivi/buildsrc/dependencies.kt

```diff
@@ -172,20 +170,32 @@ object Libs {
 
             const val runtime = "androidx.compose:compose-runtime:$version"
         }
+
+        object Hilt {
+            private const val version = "1.0.0-alpha01"
+            const val work = "androidx.hilt:hilt-work:$version"
+            const val viewmodel = "androidx.hilt:hilt-lifecycle-viewmodel:$version"
+            const val compiler = "androidx.hilt:hilt-compiler:$version"
+        }
     }
 
     object Dagger {
         private const val version = "2.28"
         const val dagger = "com.google.dagger:dagger:$version"
```

Comment:
You don't need these anymore, Hilt transitively depends on this.

Reply:
I need this for the non-Android modules unfortunately.